src/react/ui-elements/Veeam/VeeamConfiguration.test.tsx

```diff
@@ -31,25 +39,52 @@ describe('Veeam Configuration UI', () => {
       }),
   };
 
-  it('should be able to set the Veeam configuration', async () => {
-    //S
-    render(
-      <QueryClientProvider client={new QueryClient()}>
-        <Stepper
-          steps={[
+  const mockUseAccountsImplementation = () => {
+    //@ts-ignore
```

Comment:
cast `mockUseAccountsLocationsAndEndpoints` as jest.Mock to not have the ts-error anymore